ntoskrnl/io/pnpmgr/arbiters.c

```diff
@@ -331,8 +394,34 @@ IopGenericUnpackRequirement(
 {
     PAGED_CODE();
 
-    UNIMPLEMENTED;
-    return STATUS_NOT_IMPLEMENTED;
+    DPRINT("[%p] Min %I64X Max %I64X Len %X\n",
+            IoDescriptor,
+            IoDescriptor->u.Generic.MinimumAddress.QuadPart,
+            IoDescriptor->u.Generic.MaximumAddress.QuadPart,
+            IoDescriptor->u.Generic.Length);
+
+    ASSERT(IoDescriptor);
+    ASSERT(IoDescriptor->Type == CmResourceTypePort ||
+           IoDescriptor->Type == CmResourceTypeMemory);
+
+    *OutLength = IoDescriptor->u.Generic.Length;
+    *OutAlignment = IoDescriptor->u.Generic.Alignment;
+
+    *OutMinimumAddress = IoDescriptor->u.Generic.MinimumAddress.QuadPart;
+    *OutMaximumAddress = IoDescriptor->u.Generic.MaximumAddress.QuadPart;
+
+    if (IoDescriptor->u.Generic.Alignment == 0)
+        *OutAlignment = 1;
```

Comment:
Why is this needed?

Reply:
Default alignment will be set to one byte.